src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -422,6 +422,14 @@
         "resourceLink": {
             "description": "A template-able link to a resource instance. AWS-internal service links must be relative to the AWS console domain. External service links must be absolute, HTTPS URIs.",
             "$ref": "#/definitions/resourceLink"
+        },
+        "propertyTransform": {
```

Comment:
Please add clear description on what this property is used for?

Reply:
This is part of the documentation and REAME which I am planning to send out next

Reply:
My comment is to add the description similar to other properties. For example `resourceLink` property has clear description on what it is used for : "A template-able link to a resource instance. AWS-internal service links must be relative to the AWS console domain. External service links must be absolute, HTTPS URIs."